iiblib/iib_build_details_model.py

```diff
@@ -519,3 +519,52 @@ class FBCOperationsModel(IIBBuildDetailsModel):
 
     _operation_attrs = deepcopy(__slots__)
     _accepted_request_type = "fbc-operations"
+
+
+class AddDeprecationsModel(IIBBuildDetailsModel):
+    """
+    AddModel class handling data from "builds/add" endpoint, and
+    data from "builds" and "builds/<id>" IIB endpoints defined by
+    "add" request_type.
+    AddModel class inherits arguments and behaviours
```

Comment:
Please update docstring, it mentions AddModel

Reply:
Aha, nice catch! Done